scripts/install-dependencies.sh

```
@@ -0,0 +1,56 @@
+    PROJECT_ROOT="$(cd "$SCRIPT_DIR/.." && pwd)"
+    echo "📍 Detected: running from scripts folder"
+else
+    echo "❌ Error: Could not detect project structure"
+    exit 1
+fi
+
+echo "📂 Project root folder: $PROJECT_ROOT"
+
+# Install dependencies in root folder
+echo "🏠 Installing dependencies in root folder..."
+cd "$PROJECT_ROOT"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)